src/PrestaShopBundle/Resources/config/services/core/customer_service.yml

```diff
@@ -0,0 +1,19 @@
+services:
```

Comment:
This file should be moved into `src/PrestaShopBundle/Resources/config/services/core/domain/customer_service.yml
` instead
For the handler definitions, we put them in the domain subfolder (the file already exists by the way)

Reply:
Again the handlers service definition should be in the `domain` subfolder to separate them from the "regular" services (like repositories, updater, modifier, ...) which are potentially used by the handlers themselves.

But since the definition of CQRS handlers is quite verbose it's better to keep them separated.

Reply:
This file still needs to be moved in domain subfolder